Dockerfile-template.ubuntu

```diff
@@ -35,14 +35,15 @@ RUN      buildDeps="autoconf \
                     zlib1g-dev" && \
         export MAKEFLAGS="-j$(($(nproc) + 1))" && \
         apt-get -yqq update && \
-        apt-get install -yq --no-install-recommends ${buildDeps} ca-certificates && \
+        apt-get install -yq --no-install-recommends ${buildDeps} ca-certificates libgomp1 && \
```

Comment:
I added it here so that it wont be removed in the cleanup part of the script. There was complaint about missing libgomp1.so.* file when trying to start ffmpeg.